### PR TITLE
Destination Bigquery: Clean up dependencies with TD/CDK

### PR DESCRIPTION
**Pull request by gisripa**
## What
*Using TD from CDK instead of project
*Other minor dependency removed

## How
* Using features capability of cdk to add non-core dependencies.

### CODE REVIEW

**Review by edgao — APPROVED** [review]
assuming tests pass

File: airbyte-integrations/connectors/destination-bigquery/build.gradle [hunk 1]
```diff
@@ -47,12 +47,11 @@ dependencies {
     integrationTestJavaImplementation project(':airbyte-integrations:connectors:destination-bigquery')
 
     // TODO: declare typing-deduping as a CDK feature instead of importing from source.
-    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
-    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
+//    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
+//    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
```
Comment (postamar) [inline airbyte-integrations/connectors/destination-bigquery/build.gradle:51]: Just delete this, don't comment things out.

```suggestion
```

**Review by postamar — APPROVED** [review]

File: airbyte-integrations/connectors/destination-bigquery/build.gradle (hunk 1 above)
Comment (gisripa) [reply airbyte-integrations/connectors/destination-bigquery/build.gradle:51]: temp commit to verify all tests are good before deleting 😅

**Comment by gisripa** [comment]
> assuming tests pass

Failed because the cdk version it depends on didn't have my case-sensitive field fixtures i think.

**Pull request merged**